core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/testSets/person/DataFrameTests.kt

```diff
@@ -919,6 +921,15 @@ class DataFrameTests : BaseTest() {
         }
     }
 
+    @Test
+    fun `add several columns with type inference`() {
+        val f: Any = 123
+        val df = typed.add {
+            expr(infer = Infer.Type) { f } into "f"
```

Comment:
I actually expected this to be done by default. If you need to manually specify to infer types, I think people will just not do it. And then, it will be `Any`?

Reply:
It's consistent with `add` itself (default infer = Infer.Nulls) and most other usages of this enum. I think it makes sense. Looking at values to deduce type is a reflective operation and, while i didn't test it myself and decision wasn't made by me, i suppose it can be quite heavy + produce unexpected results (more precise type than needed or something). So i'd say keeping it as an option is a reasonable compromise